src/test/ui/sanitize/unsupported-target.stderr

```diff
@@ -1,4 +1,4 @@
-error: LeakSanitizer only works with the `x86_64-unknown-linux-gnu` or `x86_64-apple-darwin` target
+error: LeakSanitizer only works with the `aarch64-unknown-linux-gnu` or `x86_64-apple-darwin` or `x86_64-unknown-linux-gnu` target
```

Comment:
A better way to spell here would be "A, B or C"

Reply:
Reworded and replaced with a comma. If this is something you feel strongly about it make make sense to open an issue about it more generally. There is one implementation already in `tokens_to_string` function.